### DISCUSSION TITLE
Simple Decision: Enhancement of Proposal Visibility in Fairdrive DAO

### DISCUSSION THREAD

**Discussion by mmohorko** [fairDataSociety/Fairdrive-DAO#4] [Proposals]
Since Colony does not have notifications it's easy to overlook a proposal submitted by members. Therefore I suggest we add 1 more step into the proposal submission process that makes it obligatory for any proposal that is submitted, to notify the members too. You do this simply by posting the link to the proposal to the Telegram group and Discussions section on Github.

More information here: https://hackmd.io/@8M5lIuhKQjGXbVKpbr_rMA/rylo_GITT

Please VOTE here: https://xdai.colony.io/colony/fairdrive/decisions/tx/0xdb933bedf5bef7f7f3706ae3924c15a1b067e6d2cad72a48879cf1fe04245191

I am happy to answer any questions regarding this topic here or on the TG group.

**Comment by tfius** [comment]
or open issue here, and link to proposal on colony

**Comment by mmohorko** [comment]
This proposal was successful. The extra step in the proposal submission process was added to the docs. See step 3. here: https://github.com/fairDataSociety/Fairdrive-DAO/wiki/Proposal-and-Voting-Mechanisms
